### PR TITLE
Add example for adding lambda layer to enable bedrock API

### PR DESCRIPTION
**Pull request by jld23**
*Description of changes:*
Adding an example that explains how to add a lambda layer for bedrock access through lambda before boto3 1.28.57 (and higher) is included.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: generative-ai-solutions/bedrock-lambda-layer/lambda_base.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import logging
+import json
+import zipfile
+import io
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+layer_arn = 'arn:aws:lambda:us-east-1:141051459377:layer:bedrock-1-28-57:4'
```
Comment (celmore25) [inline generative-ai-solutions/bedrock-lambda-layer/lambda_base.py:14]: lets get rid of these ARNs before merge

**Review by celmore25 — COMMENTED** [review]
all looks good to me if we remove the ARNs :)

File: generative-ai-solutions/bedrock-lambda-layer/lambda_base.py (hunk 1 above)
Comment (jld23) [reply generative-ai-solutions/bedrock-lambda-layer/lambda_base.py]: Pushed a new commit that removed layer and role arn

**Review by celmore25 — APPROVED** [review]
good to go

**Pull request merged**